htsget.md

```diff
@@ -162,29 +175,25 @@ _optional 32-bit unsigned integer_
 </td><td>
 The start position of the range on the reference, 0-based, inclusive. 
 
-The server SHOULD respond with an `InvalidInput` error if `start` is specified and a reference is not specified
-(see `referenceName`).
+The server SHOULD respond with an `InvalidInput` error if `start` is specified and a reference is not specified (see `referenceName`).
```

Comment:
What's after changing here? I think we should keep the diff to the minimum.

Reply:
Just trying to make this section follow the same style than the rest of the document, where lines are not split.

Reply:
Ah right, that's a good idea.  It might be better to do such housekeeping stuff separately to semantic changes like this though. I know @jmarshall likes a nice clean diff!

Reply:
I haven't been able to find any other formatting issues in the whole document. Is removing 3 line breaks worth rebasing this PR and creating a new one?

Reply:
Definitely not worth another PR. Could split this into two commits (one housekeeping, one VCF) when squashing? It really makes no difference though. If nobody else complains, whatever you prefer is fine by me.